tests/test_list_accelerators.py

```diff
@@ -24,10 +26,65 @@ def test_list_accelerators_name_filter():
 
 def test_list_accelerators_region_filter():
     result = sky.list_accelerators(gpus_only=False,
-                                   clouds="aws",
+                                   clouds='aws',
                                    region_filter='us-west-1')
     all_regions = []
     for res in result.values():
         for instance in res:
             all_regions.append(instance.region)
     assert all([region == 'us-west-1' for region in all_regions])
+
+
+def test_list_accelerators_name_quantity_filter():
+    result = sky.list_accelerators(name_filter='V100', quantity_filter=4)
+    all_accelerators = []
+    for res in result.values():
+        for instance in res:
+            all_accelerators.append(
+                (instance.accelerator_name, instance.accelerator_count))
+    assert all([
+        name.__contains__('V100') and quantity == 4
+        for name, quantity in all_accelerators
+    ])
+
+
+def test_list_accelerators_positive_quantity_filter():
+    result = sky.list_accelerators(quantity_filter=4)
+    all_accelerators = []
+    for res in result.values():
+        for instance in res:
+            all_accelerators.append(instance.accelerator_count)
+    assert all(quantity == 4 for quantity in all_accelerators)
+
+
+def test_list_accelerators_name_clouds_filter():
+
```

Comment:
nit: remove empty line here and at L76. Consider adding a short comment on what the test does